gbfs.md

```diff
@@ -450,15 +431,15 @@ Example:
 
 ### free_bike_status.json
 
-Describes vehicles that are not at a station and are not currently in the middle of an active ride.
+Describes all vehicles that are not at a station and are not currently in the middle of an active ride. From v2.1-RC, also provides more information about vehicles that are at a station if the [vehicle_types.json](#vehicle_typesjson) file has been defined.
```

Comment:
Would like to clarify this definition and add line about vehicles in active rental from Best Practices. Suggesting this:

"Describes all vehicles that are not currently in active rental. Required of free floating (dockless) vehicles. Conditionally required of station based (docked) vehicles when [vehicle_types.json](#vehicle_typesjson) has been defined. Vehicles that are part of an active rental must not appear in this feed."

Reply:
This definition should also be updated under "Files" above on line 71.

Reply:
Yes that's more clear, thanks! Updated at both places.